package.json

```diff
@@ -9,9 +9,11 @@
   "dependencies": {
     "cookie-parser": "~1.4.4",
     "debug": "~2.6.9",
+    "dotenv": "^8.2.0",
```

Comment:
Is "dotenv" needed? You can get environmental variables from process.env
Example: `process.env.DB_HOST`.